lib/honey_format/converters/convert_babosa.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+BABOSA_LOADED = begin
+                  require 'babosa'
+                  true
+                rescue LoadError
+                  false
+                end
```

Comment:
Skip assigning this constant and just use `defined?(Babosa)` in the initializer instead.